coinlib/bin/build_wsl.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:io';
+
+import 'util.dart';
+
+/// Follows bitcoin-core/secp256k1's "Cross compiling" instructions.
+///
+/// Runnable in WSL.  Install the dependencies listed in the README:
+/// ```
+/// apt-get install -y autoconf libtool build-essential git cmake mingw-w64
+/// ```
+
+void main() async {
+  // Make temporary directory.
+  final workDir = Directory.current.path;
+  final tmpDir = createTmpDir();
+
+  // Clone bitcoin-core/secp256k1.
+  await execWithStdio(
+    "git",
+    ["clone", "https://github.com/bitcoin-core/secp256k1", "$tmpDir/secp256k1"],
+  );
+  Directory.current = Directory("$tmpDir/secp256k1");
+  await execWithStdio(
+    "git",
+    ["checkout", "346a053d4c442e08191f075c3932d03140579d47"],
+  );
+
+  // Build in tmpDir/secp256k1/lib.
+  Directory("lib").createSync();
+  Directory.current = Directory("lib");
+
+  // Run cmake with the provided toolchain file.
+  await execWithStdio("cmake", [
+    "..",
+    "-DCMAKE_TOOLCHAIN_FILE=../cmake/x86_64-w64-mingw32.toolchain.cmake",
+  ]);
+
+  // Build the project using "make".
+  await execWithStdio("make", []);
+
+  // Print the current dir.
+  print(Directory.current.path);
```

Comment:
I suppose this was for debugging purposes and no longer needed?

Reply:
Oh, yes, absolutely.

Reply:
Resolved by 6f620a476fcce8afb87fa35c90f81a341d886abb